sdcm/utils/user_profile.py

```diff
@@ -0,0 +1,84 @@
+from __future__ import absolute_import, annotations
```

Comment:
Please add docstrings and comments to explain this module, why it's needed, where it's used and how it can help others for future usages.

Reply:
done

Reply:
I don't see it...

Reply:
added docstrings